src/Payments.sol

```diff
@@ -289,9 +289,9 @@ contract Payments is Initializable, UUPSUpgradeable, OwnableUpgradeable, Reentra
         require(
             !settleFull || isAccountLockupFullySettled(payer),
             isBefore
-                ? "payers's account lockup target was not met as a precondition of the requested operation"
+                ? "payers's full account lockup was not met as a precondition of the requested operation"
                 :
-                "the requested operation would cause the payer's account lockup target to exceed the funds available in the account"
```

Comment:
I'm pretty sure this is incorrect in all cases since we check for this invariant in `settleAccountLockup`